controllers/controllers.go

```diff
@@ -76,6 +76,9 @@ const (
 	// FDBLocalityDCIDKey represents the key in the locality map that holds
 	// the DC ID.
 	FDBLocalityDCIDKey = "dcid"
+
+	// FDBManagedByLabel represents the label that is used to represent that the resource is managed by the operator
+	FDBManagedByLabel = "fdb-managed-by-operator"
```

Comment:
This seems to be used as a label when fetching pods in the analyze command, but it's not set by the operator.

Reply:
I think I added it in the first place but removed it from the selector again. I will remove it completely.